### ISSUE TITLE
Sentinl-v5.6.5 configuration issue

### ISSUE THREAD

**Issue by neelrana** [sirensolutions/sentinl#279]
_1. Issue Description_

Hi, I have a configuration issue with sentinl version 5.6.5 with ELK (elasticsearch + logstatsh + kibana) 5.6.5.

_2. Reproducing the Issue step-by-step_

I followed through the following steps to install sentinl:

a) /usr/share/kibana/bin/kibana-plugin install https://github.com/sirensolutions/sentinl/releases/download/tag-5.6.2/sentinl-v5.6.5.zip

b) git clone https://github.com/sirensolutions/sentinl

c) cd sentinl
    npm install
    gulp package --version=5.6.5

Then I restart the kibana service and I get this error:

![screenshot from 2017-12-14 16-44-40](https://user-images.githubusercontent.com/9300684/34003814-2d62e2d4-e0ee-11e7-82a6-9a6558b4bc4d.png)

![screenshot from 2017-12-14 16-44-47](https://user-images.githubusercontent.com/9300684/34003819-30362de0-e0ee-11e7-8c4c-8134045eefe7.png)

Any ideas of what it could be? I tried Sentinl-v6.0.0 with ELK 6.0.0 and was getting the similar problem (see the other issue thread). Thus, I reverted back to version 5.6.5. Is there anything that I am doing wrong? Please let me know. Please help.

Thanks

**Comment by lmangani** [comment]
Please use a ZIP [release](https://github.com/sirensolutions/sentinl/releases) for your version. This is the supported installation method. Feel free to comment if still needed.

**Comment by sergibondarenko** [comment]
Hi @neelrana 
Please check the installation instruction update https://github.com/sirensolutions/sentinl/wiki/SENTINL-Installation#kibana-5x

**Comment by neelrana** [comment]
@lmangani and @sergibondarenko Thank you for your reply. I reinstalled Sentinl following the steps shown in the wiki under user mode. However, I still get the same issue.

I did `grep version package.json` and I get:
```
   "version": "5.6.4-SNAPSHOT",
    "version": "5.6.5"
```
I understand that sentinl and kibana versions need to be the same. But I dont understand the "_"version": "5.6.4-SNAPSHOT",_". Is this the problem? or could it be something else?
